models/nexusmutual/ethereum/covers/nexusmutual_ethereum_product_information.sql

```diff
@@ -0,0 +1,16 @@
+{{ config(
+    alias = 'product_information',
+    materialized = 'view',
+    post_hook='{{ expose_spells(\'["ethereum"]\',
```

Comment:
do all spells in this PR need to be exposed on data explorer? or can we narrow down to just a few?

Reply:
updated 2 old ones, all the rest - yes please